psychopy/tests/test_misc/test_event.py

```diff
@@ -226,6 +227,20 @@ def setup_class(self):
         self.win = Window([128,128], monitor=mon, winType='pyglet', pos=[50,50], autoLog=False)
         assert pygame.display.get_init() == 0
 
+    def test_shutdown_key(self):
+        key = 'escape'
```

Comment:
Ok so there's less magic involved than I hoped:
```python
In [1]: from psychopy.preferences import prefs

In [2]: prefs.general['shutdownKeys'] = 'foo'

In [3]: prefs.general['shutdownKeys']
Out[3]: 'foo'
```
Hence I guess the test is not working properly.